### PR TITLE
Bundle event emitter with lib

### PR DESCRIPTION
**Pull request by robgonnella**
We now bundle event emitter with the library for use in web "module"
contexts.

### CODE REVIEW

**Comment by robgonnella** [comment]
Closes: https://github.com/robgonnella/redux-analytics-manager/issues/12

File: package.json [hunk 1]
```diff
@@ -17,35 +17,22 @@
     "analytics",
     "analytics-manager"
   ],
-  "main": "commonjs/index.js",
-  "module": "module/index.js",
```
Comment (jorgecasar) [inline package.json:21]: @robgonnella deleting module property won't support module and tools like @pikapkg or bundlephobia work properly in your package

**Comment by jorgecasar** [comment]
`module` property in package.json is required for so many tools to know that supports a ES module

File: package.json (hunk 1 above)
Comment (robgonnella) [reply package.json:21]: Once again thanks for reporting! Just added module support back and published 2.1.5
https://github.com/robgonnella/redux-analytics-manager/commit/ebc014185fae130ad49f312a1801a5c7e63b0b7b

**Pull request merged**